src/pyhf/optimize/opt_minuit.py

```diff
@@ -84,19 +83,29 @@ def _minimize(
         underlying minimizer.
 
         Minimizer Options:
-            maxiter (`int`): maximum number of iterations. Default is 100000.
-            return_uncertainties (`bool`): Return uncertainties on the fitted parameters. Default is off.
+            maxiter (`int`): maximum number of iterations. See :attr:`~iminuit.Minuit.ncalls`. Default is ``100000``.
+            tol (`float`): tolerance for convergence. See :attr:`~iminuit.Minuit.tol`. Default is ``0.1``.
+            strategy (`int`): current minimization strategy. See :attr:`~iminuit.Minuit.strategy`. Default is ``1``.
+            return_uncertainties (`bool`): Return uncertainties on the fitted parameters. Default is off (``False``).
+            return_correlations (`bool`): Return correlations of the fitted parameters. Default is off (``False``).
```

Comment:
These are nice docstrings, so it is kinda too bad they don't show up in the docs render. I'm not sure where they would show up as this is a private method, but these `kwargs` would be nice to make clear to users.

Reply:
Migrated the last two options here up to `mixin.minimize` so it will show up in documentation.